LICENSE.txt

```diff
@@ -1,27 +1,30 @@
-Copyright (c) 2015-2022, conda-forge
```

Comment:
Please do not modify this file as it is specific to conda-forge and not this package.